cocotb/share/lib/simulator/simulatormodule.cpp

```diff
@@ -923,62 +923,138 @@ PyMODINIT_FUNC PyInit_simulator(void)
 
 static PyMethodDef gpi_sim_hdl_methods[] = {
     {"get_signal_val_long",
-        (PyCFunction)get_signal_val_long, METH_NOARGS,
-        "Get the value of a signal as a long"},
+        (PyCFunction)get_signal_val_long, METH_NOARGS, PyDoc_STR(
+            "get_signal_val_long($self)\n"
+            "--\n\n"
+            "Get the value of a signal as a long"
+        )
+    },
     {"get_signal_val_str",
-        (PyCFunction)get_signal_val_str, METH_NOARGS,
-        "Get the value of a signal as an ASCII string"},
+        (PyCFunction)get_signal_val_str, METH_NOARGS, PyDoc_STR(
+            "get_signal_val_str($self)\n"
+            "--\n\n"
+            "Get the value of a signal as an ASCII string"
```

Comment:
```suggestion
            "Get the value of a signal as an string"
```

Reply:
Typo in the suggestion: "Get the value of a signal as a string"